processor_function/shared/cnab_builder.py

```diff
@@ -113,9 +116,31 @@ def deploy_aci(self):
 
         credential = AzureIdentityCredentialAdapter()
         aci_client = ContainerInstanceManagementClient(credential, self._subscription_id)
+
+        service_bus = ServiceBus()
+        service_bus.send_status_update_message(self._id, "ACI Deployment started", "Deploying ACI container: " + self._container_group_name)
```

Comment:
I think we should make a Status Enum(which can be a copy of Staus Enum in API)

Reply:
Also strings should be put into resources and used with a well defined NAME_LIKE_THIS. Right now it is very hard to read.

Reply:
Added strings; let's split enum into own task -> don't want to copy-paste code from API, and requires alignment on what values should actually be there (how granular the status should be reported)